File: IT/2/Interfacce grafiche/Esercizi/Java/Esercizio4.java
/* file Esercizio4.java */

import java.awt.Dimension;
import javax.swing.JFrame;
import javax.swing.ImageIcon;

public class Esercizio4 {
	public static void main(String [] args) {
		JFrame frame = new JFrame();

		frame.setSize(new Dimension(150, 100));

		ImageIcon icon = new ImageIcon("img/icon.gif");
		frame.setIconImage(icon.getImage());

		frame.setTitle("Icona");
		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
		frame.setVisible(true);
	}
}

File: IT/2/Interfacce grafiche/Esercizi/Python/esercizio4.py
# file esercizio4.py

try:
	from tkinter import Tk, Frame, PhotoImage
except ImportError:
	from Tkinter import Frame, Label, PhotoImage

frame = Frame(height=100, width=150)

img = PhotoImage(file="img/icon.gif")

frame.tk.call("wm", "iconphoto", frame.winfo_toplevel(), img)
frame.master.wm_title("Icona")
frame.pack()
frame.mainloop()

